sdk/trace/batch_span_processor.go

```diff
@@ -201,10 +201,9 @@ func (bsp *batchSpanProcessor) ForceFlush(ctx context.Context) error {
 			}
 		}
 
-		wait := make(chan error)
+		wait := make(chan error, 1)
 		go func() {
 			wait <- bsp.exportSpans(ctx)
-			close(wait)
 		}()
```

Comment:
Hi! Came here from the release page. I wonder if this even needs to run in its own goroutine? Is `exportSpans()` supposed to stop when `ctx` is done? Wouldn't it be a bug/leak if it doesn't stop?

Reply:
I created https://github.com/open-telemetry/opentelemetry-go/pull/6416